File: chessgame-api/src/main/java/git/dimitrikvirik/chessgameapi/repository/GameRepository.java
package git.dimitrikvirik.chessgameapi.repository;

import git.dimitrikvirik.chessgameapi.model.domain.Game;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository

public interface GameRepository extends JpaRepository<Game, Long> {
}

File: chessgame-api/src/main/kotlin/git/dimitrikvirik/chessgameapi/repository/redis/GameRepository.kt
package git.dimitrikvirik.chessgameapi.repository.redis

import git.dimitrikvirik.chessgameapi.model.redis.Game
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface GameRepository : CrudRepository<Game?, String?> {

}